lib/hoge.go

```diff
@@ -1,6 +1,8 @@
 package lib
 
+import "github.com/crifff/private"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/crifff/private (can't find import: )
* [errcheck] error return value not checked (could not import github.com/crifff/private (cannot find package "github.com/crifff/private" in any of:)
* [interfacer] could not import github.com/crifff/private (cannot find package "github.com/crifff/private" in any of:
* [aligncheck] could not import github.com/crifff/private (cannot find package "github.com/crifff/private" in any of:
* [unconvert] could not import github.com/crifff/private (cannot find package "github.com/crifff/private" in any of:
* [structcheck] unused struct field could not import github.com/crifff/private (cannot find package "github.com/crifff/private" in any of:
* [varcheck] unused global variable could not import github.com/crifff/private (cannot find package "github.com/crifff/private" in any of:

<!-- Comment by [SideCI](https://sideci.com) -->